lib/Dialect/Arc/Transforms/StripSV.cpp

```diff
@@ -9,6 +9,7 @@
 #include "circt/Dialect/Arc/ArcOps.h"
 #include "circt/Dialect/Arc/ArcPasses.h"
 #include "circt/Dialect/Comb/CombOps.h"
+#include "circt/Dialect/Emit/EmitDialect.h"
```

Comment:
Don't we need to link `ArcTransforms` against `CIRCTEmit`?

Reply:
Yes, good catch. It did not fail on my builds, but that's probably coincidental.